### ISSUE TITLE
conflict when import com.websudos.phantom.Implicits._

### ISSUE THREAD

**Issue by graph1zzlle** [websudos/phantom#195]
Running into this issue, using Scala 2.11.4 and so phantom-dsl_2.11.

```scala
[error]  found   : String
[error]  required: ?{def nonEmpty: ?}
[error] Note that implicit conversions are not applicable because they are ambiguous:
[error]  both method augmentString in object Predef of type (x: String)scala.collection.immutable.StringOps
[error]  and method wrapString in class LowPriorityImplicits of type (s: String)scala.collection.immutable.WrappedString
[error]  are possible conversion functions from String to ?{def nonEmpty: ?}
[error]               .filter(row ⇒ row.getString("someColumn").nonEmpty)
[error]                                          ^
```

Already seen https://github.com/websudos/phantom/issues/145, seems like not everything is fixed ?

**Comment by alexflav23** [comment]
@graph1zzlle Please correct me if I am wrong but what you are looking for here is a conversion from ```String``` to an ops wrapper, which I believe doesn't have a whole lot to do with phantom, but rather with a Scala clash. ```LowPriorityImplicits``` was removed by Scala 2.11, I don't understand how it winds up in the classpath.

We will investigate further and see what's causing the import.

**Comment by graph1zzlle** [comment]
Maybe I am wrong, I will do more investigation and report back.

**Comment by alexflav23** [comment]
@graph1zzlle Any news on this? Just want to double check before closing the issue. Many thanks!

**Comment by alexflav23** [comment]
@graph1zzlle This has definitely been resolved in the upcoming 1.6.0 and I've closed the ticket as such. The ```Implicits``` object has been deprecated in a favour of a more Scala friendly ```package object```, and there's no sign of the conflict you mention.

**Comment by D-Roch** [comment]
@alexflav23 do you know when it will be released on maven ?
